bin/7z

```diff
@@ -1,3 +0,0 @@
-#!/usr/bin/env bash
```

Comment:
I still use in the release pipeline, so please revert

Reply:
No problem at all! Reinstated and re-pushed at ff4c8175fe079d8502b82864fc3c6cd71bfa69d6.